recipe/meta.yaml

```diff
@@ -26,12 +27,14 @@ requirements:
     - packaging >=17.1
     - numpy >=1.17.2
     - pyyaml >=5.4
-    - pytorch >=1.10.0
+    - pytorch >=1.11.0
+    - future >=0.17.1
```

Comment:
I'm not sure if `future` is really required https://github.com/Lightning-AI/lightning/blob/2.0.2/requirements/pytorch/base.txt
It's mentioned in https://github.com/Lightning-AI/lightning/blob/2.0.2/environment.yml but I think it's for development purposes

Reply:
it's not. I removed it.